### DISCUSSION TITLE
Extract mobtime server from code

### DISCUSSION THREAD

**Discussion by ethanstrominger** [mobtimeapp/mobtime#291] [Ideas]
I see this was started here: https://github.com/mobtimeapp/mobtime-nodejs.  What is the status?

We (Joel Siblerman and I) have created a mobtimer with server, frontend, and common code separated - eventually may split into repositories.  Still a WIP but close at https://github.com/jcs-instructor/mobtimer.  Front end is in react - tried to use your frontend but there were npm conflicts that prevented it from compiling.

**Comment by mrozbarry** [comment]
Right now, [this](https://github.com/mobtimeapp/mobtime) is the main repo that has both frontend and backend that are currently in production. The mobtimeapp/mobtime-nodejs is an ongoing effort of upgrading and simplifying things, as well as a bit of an experiment on building a laravel-like server in nodejs.

Both fortunately (for me) and unfortunately (for the mobbing community using mobtime), my wife and I just had our second baby, which means I've got even less time than normal while we are adjusting.

**Comment by ethanstrominger** [comment]
Congratulations!

Hmm…we (Joel and I) are also trying to create a simpler app and have split up front end and backend.  We are planning on adding more features - separate integrated break timer/stretch reminder, chat, raise hand, and probably more.  There may be cooperation opportunities?  Or you want to keep the features basic?

FYI, when we tried to create a simple FERP application there was some error related to compatibility with other packages.  Rather than try to solve it, we decided to use React as I am familiar with it and there is a large user community.
